sig-platform/OWNERS

```diff
@@ -1,7 +1,12 @@
 reviewers:
-  - add-names-here
+  - VaishnaviHire
+  - Xaenalt
+  - atheo89
+  - JooHo
 approvers:
-  - add-names-here
+  - LaVLaS
+  - samuelvl
+  - gabe-l-hart
+  - taneem-ibrahim
 labels:
-  - sig/platform
-
+  - sig/platform
```

Comment:
Missing newline? :-)

Reply:
Hi @anishasthana Sorry I see it with a new line in my editor. What exactly is missing here?

Reply:
That's weird. Here's the diff I'm seeing in the GitHub UI 
<img width="289" alt="image" src="https://user-images.githubusercontent.com/6526150/186171563-92f57bdd-f250-47a5-bd6f-0ecb17688bfa.png">